### ISSUE TITLE
Routine releases in Sprint 38

### ISSUE THREAD

**Issue by bassosimone** [ooni/probe#1439]
- [x] probe-cli: take a look at [go report card](https://goreportcard.com/report/github.com/ooni/probe-cli)
- [x] psiphon: run ./update.bash
- [x] probe-cli: address any outstanding TODO in the diff since last release (or create an issue for it)
- [x] probe-cli: update dependencies
- [x] probe-cli: update internal/engine/httpheader/useragent.go
- [ ] probe-cli: update internal/version/version.go
- [x] probe-cli: update github.com/ooni/probe-assets dependency
- [x] probe-cli: update bundled certs (using `go generate ./...`)
- [ ] probe-cli: make sure all workflows are green
- [ ] probe-cli: `go test -race -count 1 ./...` must pass locally
- [ ] probe-cli: tag a new version
- [ ] probe-cli: update internal/version/version.go to be alpha
- [ ] probe-cli: create release at GitHub
- [x] probe-android: pin to latest oonimkall
- [x] probe-ios: pin to latest oonimkall
- [ ] probe-desktop: pin to latest cli
- [ ] probe: create issue for next routine release
- [ ] e2etesting: see whether we can remove legacy checks

**Comment by bassosimone** [comment]
I am going to call this effort done. We had lots of churn in the build system recently. This was due to the need to work around encryption algorithm selection issues as well as the need to produce arm/arm64 binaries. Both things have been done and, in the context of this task, I was able to produce a bunch of release candidates. In a few minutes, I'll produce the latest release candidate. We will continue the work of releasing in the next Sprint with a new issue (https://github.com/ooni/probe/issues/1468).
